src/Components/RelevanceInspectorResult.tsx

```diff
@@ -32,6 +23,8 @@ export default class RelevanceInspectorResult extends React.Component<
   private headlessRelevanceInspector: RelevanceInspectorType;
   private result: Result;
   private index: number;
+  private _isMounted = false;
```

Comment:
🤔 
Personally I can remove all the isMounted stuff and my codesandbox builds just fine:
https://codesandbox.io/s/objective-zhukovsky-y8ugo?file=/src/Components/RelevanceInspectorResult.tsx